### PR TITLE
Update mobile-compile-yourself.rst

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/mobile/mobile-compile-yourself.rst [hunk 1]
```diff
@@ -33,3 +33,17 @@ To deploy in production with privately-hosted mobile apps compiled by your organ
 .. image:: ../images/mobile_push_contents.png
 
 6. Deploy to an Enterprise App Store
+
+Commands to Build the App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The set of commands for building the app are used in conjunction with Fastlane.  For most of them, you will need to modify the Fastfile in order to make it work since they are very tightly coupled with Mattermost's build and deployment process.
+
+You will **always** be able to build an unsigned version of the app as it does not need provisioning profiles or certificates as long as you set up *Fastlane* in your environment.
```
Comment (jasonblais) [inline source/mobile/mobile-compile-yourself.rst]: We usually reserve bolding for [field names](https://docs.mattermost.com/process/documentation-guidelines.html#text-formatting), so we can remove it for `always`. Also, `Fastlane` should either be italicized throughout the section or not for consistency.

We can also shorten this sentence a little, maybe:

```
If you set up Fastlane, no provisioning profiles or certificates are needed to build an unsigned version of the app.
```

^should make sure the content stays the same after rephrasing.

File: source/mobile/mobile-compile-yourself.rst [hunk 2]
```diff
@@ -33,3 +33,17 @@ To deploy in production with privately-hosted mobile apps compiled by your organ
...
+
+ - **make build-ios**: Builds the iOS app and generates the *.ipa* file to be distributed. This make command expects an argument as the target which can be *dev*, *beta* or *release*. Depending on the target, a Fastlane script runs and each lane has the appropriate certificates and steps according to the Mattermost release process.
```
Comment (jasonblais) [inline source/mobile/mobile-compile-yourself.rst]: `each lane` --> what is a lane?

File: source/mobile/mobile-compile-yourself.rst [hunk 3]
```diff
@@ -33,3 +33,17 @@ To deploy in production with privately-hosted mobile apps compiled by your organ
...
+ - **make build-android**: Builds the Android app and generates the *.apk* file to be distributed. This make command expects an argument as the target which can be *dev*, *alpha* or *release*. Depending on the target, a Fastlane script runs and each lane has the appropriate certificates and steps according to the Mattermost release process.
+ - **make unsigned-ios**: Builds the iOS app and generates an unsigned *Mattermost-unsigned.ipa* file in the project's root directory.
+ - **make unsigned-android**: Builds the Android app and generates an unsigned *Mattermost-unsigned.apk* file in the project's root directory.
+
+If you plan to use the **make build-*** commands, be sure to modify Fastlane to suit your needs or the commands will fail.
```
Comment (jasonblais) [inline source/mobile/mobile-compile-yourself.rst]: `be sure to modify Fastlane to suit your needs or the commands will fail.` --> is there an example we can give?

**Comment by amyblais** [comment]
Not sure, I'll probably add these to the [first PR](https://github.com/mattermost/docs/pull/1627) I opened and close this one.

**Pull request closed without merging**